prov/sockets/src/sock_dom.c

```diff
@@ -519,8 +519,8 @@ int sock_domain(struct fid_fabric *fabric, struct fi_info *info,
 	sock_domain->dom_fid.ops = &sock_dom_ops;
 	sock_domain->dom_fid.mr = &sock_dom_mr_ops;
 
-	if (!info || !info->domain_attr ||
-	    info->domain_attr->data_progress == FI_PROGRESS_UNSPEC)
+	if (!info->domain_attr ||
+		info->domain_attr->data_progress == FI_PROGRESS_UNSPEC)
 		sock_domain->progress_mode = FI_PROGRESS_AUTO;
```

Comment:
The alignment is off.  4 spaces were replaced by a tab, which indents is second line of the if statement too far.